src/android/java/io/jxcore/node/BtConnectorHelper.java

```diff
@@ -49,15 +52,18 @@ public BtConnectorHelper() {
         this.context = jxcore.activity.getBaseContext();
     }
 
-
     public BTConnector.WifiBtStatus Start(String peerName,int port){
```

Comment:
I made them not to use synchronized keyword. Basically with jxcore they should be always called from the same thread. I did though use local variables there to make the desing thread safe